packages/workspace/src/tasks-runner/utils.ts

```diff
@@ -152,3 +149,19 @@ function unparseOption(key: string, value: any, unparsed: string[]) {
     unparsed.push(`--${key}=${value}`);
   }
 }
+
+function interpolate(template: string, data: any): string {
+  template = template.replace(/{([\s\S]+?)}/g, function (match: string) {
```

Comment:
don't reassign, return instead

use lambdas instead of function

Reply:
Done